postgres/terraform/aws/variables.tf

```diff
@@ -21,3 +21,9 @@ variable "role_name" {
   type = string
   default = "postgres"
 }
+
+variable "force_destroy_bucket" {
+  type        = bool
+  default     = false
```

Comment:
this should still default to true, but we should override it in our cluster's installation

Reply:
done